Exception handling.js

```diff
@@ -1,19 +1,19 @@
-
-
-try {
-    let a = 3;
-    console.log(a);
-
-
-} catch(e) {
-    alert('let перед использованием нужно объявить')
-
-}
-
-try{
-    let b=1/0
-    console.log(b)
-
-}catch (e) {
-    alert('на ноль делить нельзя');
-}
+
+
+try {
+    let a = 3;
+    console.log(a);
+
+
+} catch(e) {
+    alert('let перед использованием нужно объявить')
+
+}
+
+try{

```

Comment:
You don't generate an Error here, 1/0 = Infinity in JS. You need to generate Error with throw new Error

Reply:
No, this is not correct, there is no error generation here.